### PR TITLE
FAQs page

### PR DESCRIPTION
**Pull request by jamesatheyDDS**
Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

- [x] run the application locally (`bin/rails server`) and verified that my changes behave as expected.
- [x] run static code analysis (`bin/rubocop`) and vulnerability scan (`bin/brakeman`) against my changes.
- [x] run the test suite (`bin/rake spec`) and verified that all tests pass.
- [ ] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [ ] thoroughly outlined below the steps necessary to test my changes.
- [ ] attached screenshots illustrating relevant behavior before and after my changes.
- [ ] read, understand, and agree to the terms described in [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md).
- [ ] added my name, email address, and copyright date to [CONTRIBUTORS.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTORS.md).

## Summary of Changes

This pull request…

- _Summarize change number one._
- _Summarize change number two._
- _etc. etc. etc._

## Testing

To verify the changes proposed in this pull request…

1. _Describe testing step one._
1. _Describe testing step two._
1. _etc. etc. etc._

## Screenshots

_Attach relevant before and after screenshots here._

### CODE REVIEW

**Comment by jamesatheyDDS** [comment]
No content in the FAQs yet, and the checkboxes are not in columns. At some point, I should try using a search box instead of checkboxes.

File: app/views/pages/faqs.html.erb [hunk 1]
```diff
@@ -6,6 +6,270 @@
...
+    </div>
+
+    <ul class="usa-accordion-bordered" id="faq-accordions">
+
+      <li class="faq-accordion" data-categories="claims">
+        <button class="usa-accordion-button faq-accordion-button" aria-controls="faq-claim-limit-content">
+          How long do I have to file a claim?
+        </button>
+        <div id="faq-claim-limit-content" class="usa-accordion-content">
+        </div>
```
Comment (jgarber623-gov) [inline app/views/pages/faqs.html.erb]: Suggest adding `<%# TODO Add FAQ content %>` in these empty `<div>`s so they show up when running `bin/rails notes`.

File: app/views/pages/faqs.html.erb [hunk 2]
```diff
@@ -6,6 +6,270 @@
...
-    <%# TODO Add page content %>
+    <%# TODO Hide the checkboxes when the screen is too narrow %>
+    <div class="usa-grid" id="faq-filters">
+
+      <div class="usa-width-one-third">
+        <fieldset class="usa-fieldset-inputs usa-sans">
+          <ul class="usa-unstyled-list">
+            <li>
+              <input id="claims-checkbox" type="checkbox" value="claims">
+              <label for="claims">Claims</label>
```
Comment (jgarber623-gov) [inline app/views/pages/faqs.html.erb]: The `for` attribute's value should match the `<input>`'s `id` that the label is associated with. In this case, `<label for="claims-checkbox">`.

Comment (jgarber623-gov) [inline app/views/pages/faqs.html.erb]: @jamesatheyDDS Related to this, I noticed the UI doesn't match [the USWDS' checkbox form controls demo page](https://standards.usa.gov/components/form-controls/#checkboxes). Long, dumb story short: I made a mistake in v1.3.0 of the uswds-rails gem and it needs updating:

```bundle update uswds-rails --full-index```

With the update, you'll get the proper UI and (with the updated `for` attribute values) you should still get the desired behavior.

File: app/views/pages/faqs.html.erb [hunk 3]
```diff
@@ -6,6 +6,270 @@
...
+      // no tags matched an enabled category, so hide the accordion
+      acc.style.display = "none";
+  }
+}
+
+for (i = 0; i < tagcheckboxes.length; i++) {
+  tagcheckboxes[i].onclick = checkboxOnClick;
+}
+
+</script>
```
Comment (jgarber623-gov) [inline app/views/pages/faqs.html.erb]: I'm going to work up a PR with a sample JavaScript pattern we can use across the site for building out functionality like this.

File: app/views/pages/faqs.html.erb [hunk 4]
```diff
@@ -6,6 +6,270 @@
   <div class="usa-width-three-fourths usa-layout-docs-main_content">
     <h1><%= @page_title %></h1>
 
-    <%# TODO Add page content %>
+    <%# TODO Hide the checkboxes when the screen is too narrow %>
+    <div class="usa-grid" id="faq-filters">
+
+      <div class="usa-width-one-third">
```
Comment (jordandds) [inline app/views/pages/faqs.html.erb]: Why not use a (mildly) more semantic HTML5 tag here? Something like `<section>`? Note: a valid reason may be: we don't use them anywhere else and it's `div`s all the way down.

File: app/views/pages/faqs.html.erb [hunk 5]
```diff
@@ -6,6 +6,270 @@
...
+        </button>
+        <div id="faq-claim-limit-content" class="usa-accordion-content">
+        </div>
+      </li>
+
+      <li class="faq-accordion" data-categories="scheduling">
+        <button class="usa-accordion-button faq-accordion-button" aria-controls="faq-pickup-date-content">
+          Can I change my pick-up date after the pre-move survey has been done?
+        </button>
+        <div id="faq-pickup-date-content" class="usa-accordion-content">
```
Comment (jordandds) [inline app/views/pages/faqs.html.erb]: If you go with HTML5 tags then I would suggest using a `main` here, or if you use a `main` hight up the tree, then perhaps a `section` tag.

File: app/views/pages/faqs.html.erb [hunk 6]
```diff
@@ -6,6 +6,270 @@
...
+          </ul>
+        </fieldset>
+      </div>
+
+    </div>
+
+    <ul class="usa-accordion-bordered" id="faq-accordions">
+
+      <li class="faq-accordion" data-categories="claims">
+        <button class="usa-accordion-button faq-accordion-button" aria-controls="faq-claim-limit-content">
```
Comment (jordandds) [inline app/views/pages/faqs.html.erb]: Should these be `button`s? Why not a `header`? (or `h3/4/5`?)

File: app/views/pages/faqs.html.erb (hunk 3 above)
Comment (jordandds) [inline app/views/pages/faqs.html.erb]: +1000 More specifically, I'd like to see this code in an IIFE and in its own `.js` file.

File: app/views/pages/faqs.html.erb (hunk 5 above)
Comment (jgarber623-gov) [inline app/views/pages/faqs.html.erb]: @jordandds We're already using `<main>` in [the application layout](https://github.com/deptofdefense/move.mil/blob/master/app/views/layouts/application.html.erb#L113-L115).

File: app/views/pages/faqs.html.erb (hunk 6 above)
Comment (jgarber623-gov) [inline app/views/pages/faqs.html.erb]: I did a quick test on this and there's nothing in the USWDS that would prohibit use of any other (potentially more appropriate) element.

**Pull request merged**